src/actions/config.js

```diff
@@ -44,35 +49,54 @@ export function asyncConfigInit() {
   };
 }
 
-export function asyncConfigFetch(configUrlList) {
+export function asyncConfigFetch() {
   return dispatch => {
     dispatch(configFetch());
     let opts = {};
     opts.headers = { Accept: 'application/javascript' };
-    let configUrl = configUrlList.shift();
+    let configUrl = './config.js';
     http.get(configUrl, opts, function(error, response) {
       if (response) {
         dispatch(configFetchSuccess());
         try {
-          let userConfig = JSON.parse(response.text);
-          Object.keys(userConfig).map(function(key) {
-            dispatch(configUpdateByKey(key, userConfig[key]));
+          let config = JSON.parse(response.text);
+          Object.keys(config).map(function(key) {
+            dispatch(configUpdateByKey(key, config[key]));
           });
-          if (configUrlList.length > 0) {
-            dispatch(asyncConfigFetch(configUrlList));
-          } else {
-            dispatch(asyncIntlFetchTranslations());
-          }
         } catch (e) {
           dispatch(notificationAddError(e.message + ' ' + configUrl));
         }
+        dispatch(asyncUserConfigFetch());
       } else {
         dispatch(configFetchError());
       }
     });
   };
 }
 
+export function asyncUserConfigFetch() {
+  return dispatch => {
+    dispatch(configFetch());
+    let opts = {};
+    opts.headers = { Accept: 'application/javascript' };
+    let configUrl = './userConfig.js';
```

Comment:
I think we should stick with the current approach.  WordPress "firewalls" should be configurable such that the user can unblock our requests.  I don't want to cater to WordPress's poor development practices here.